CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+<p align="center">
+  <a href="https://snek.at/" target="_blank" rel="noopener noreferrer">
+    <img src="https://avatars2.githubusercontent.com/u/55870326?s=400&u=c6c7f06305ddc94747d474850fde7b2044f53838&v=4" alt="SNEK Logo" height="150">
+  </a>
+</p>
+
+<h3 align="center">Official Contributing Guidelines</h3>
+
+<p align="center">
+  A set of rules outlining the norms, rules, and responsibilities of,
+  and or proper practices for, an individual.
+</p>
+
+## Table of contents
+
+-   [Table of contents](#table-of-contents)
+-   [Commits](#commits)
+-   [Pull Requests](#pull-requests)
+
+## [](#commits)Commits
+
+-   All commits may start with an git-cz type
```

Comment:
link to https://github.com/streamich/git-cz

Reply:
```suggestion
-   All commits may start with an [git-cz](https://github.com/streamich/git-cz) type
```